src/sql/azdata.d.ts

```diff
@@ -3291,6 +3291,8 @@ declare module 'azdata' {
 		valueType: DeclarativeDataType;
 		isReadOnly: boolean;
 		width: number | string;
+		showCheckAll?: boolean;
```

Comment:
These should go in azdata.proposed.d.ts

You can just declare the interface like this and it'll merge the two together

```
export interface DeclarativeTableColumn {
		showCheckAll?: boolean;
		isChecked?: boolean;
}
```